docs/developer-guide/plugins.md

```diff
@@ -34,7 +34,8 @@ const meta = {
   url: "https://github.com/foo-org/stylelint-selector-no-foo/blob/main/README.md"
 };
 
-const ruleFunction = (primary, secondaryOptions) => {
+/** @type {import('stylelint').Rule} */
```

Comment:
Should this also demonstrate the generic types that can be declared?

Reply:
Right, the `Rule` type can receive type parameters for primary and secondary options. But I think simplifying the code example is better here so that people can quickly understand it.

Reply:
I agree